app/views.py

```diff
@@ -1,22 +1,54 @@
-from flask import render_template, request
+from flask import render_template, request, redirect, url_for
 
 from app import app
 
+users = set()
+users_bucketlist = []
+
 @app.route('/')
 def index():
     return render_template('home.html')
 
-@app.route('/login')
-def login():
-    return render_template('login.html')
 
-@app.route('/signup')
+
+@app.route('/signup', methods=['GET', 'POST'])
 def signup():
+    if request.method == 'POST':
+        username = str(request.form.get('inputName'))
+        email = str(request.form.get('inputEmail'))
+        password = str(request.form.get('inputPassword'))
+
+        if username:
+            users.add(username)
+            users.add(email)
+            users.add(password)
+            print users
```

Comment:
Are you checking the edge case where a user doesn't populate email and password in their form. Adding all the details the user provides in a set makes an implication that two people cannot have the same bucketlist, I would recommend using objects.